pkg/apis/minio.min.io/v2/helper_test.go

```diff
@@ -338,8 +338,8 @@ func TestTenant_GetDomainHosts(t1 *testing.T) {
 					Features: &Features{
 						Domains: &TenantDomains{
 							Minio: []string{
-								"domain1.com:8080",
-								"domain2.com",
+								"https://domain1.com:8080",
```

Comment:
I am not sure I still understand why do we need to do this?

We should not allow URL endpoints in domain inputs - and allowing that is making it multiple different ways to specify things.

If tenant is TLS we want them to use TLS for their domains as well, allowing them to configure non TLS based scheme for incoming domain is not useful IMO.

We did make a mistake previously and that should be removed.

Reply:
@dvaldivia we should break such users and I honestly don't think anyone would be doing that. If they did we should let them fix it and it takes only few seconds for them to address it

Reply:
@harshavardhana As requested, latest commit does not allow URL endpoints in domain inputs

Reply:
After discussion, it was agreed that if the user omits schema then assume https if the tenant uses TLS; else http if non-TLS
If the schema is explicit then use that. This is the current fix.
A follow up UI UX meeting is needed for further enhancement, beyond solving the current bug.